code/modules/antagonists/nukeop/clownop.dm

```diff
@@ -5,22 +5,28 @@
 	antagpanel_category = "ClownOp"
 	nukeop_outfit = /datum/outfit/syndicate/clownop
 
-/datum/antagonist/nukeop/leader/clownop
-	name = "Clown Operative Leader"
-	roundend_category = "clown operatives"
-	antagpanel_category = "ClownOp"
-	nukeop_outfit = /datum/outfit/syndicate/clownop/leader
-	challengeitem = /obj/item/nuclear_challenge/clownops
+/datum/antagonist/nukeop/clownop/admin_add(datum/mind/new_owner,mob/admin)
+	new_owner.assigned_role = "Clown Operative"
+	new_owner.add_antag_datum(src)
+	message_admins("[key_name_admin(admin)] has clown op'ed [key_name_admin(new_owner)].")
+	log_admin("[key_name(admin)] has clown op'ed [key_name(new_owner)].")
 
 /datum/antagonist/nukeop/clownop/apply_innate_effects(mob/living/mob_override)
+	..()
```

Comment:
```suggestion
	. = ..()
```
Always preserve the return value (even if it's just `null`) unless you have a specific reason not to.